catalog/app/containers/Bucket/errors.js

```diff
@@ -29,6 +29,16 @@ export class FileNotFound extends BucketError {}
 
 export class VersionNotFound extends BucketError {}
 
+export class ManifestTooLarge extends BucketError {
+  constructor(props) {
+    super(
+      `Package manifest at s3://${props.bucket}/${props.key} is too large: ${props.actualSize} (max size: ${props.maxSize})`,
+      props,
+    )
+  }
+}
+ManifestTooLarge.displayName = 'ManifestTooLarge'
```

Comment:
We should use a different name for this, to stop confusing ESLint :)

Reply:
what's wrong with it? i didnt see any errors / warnings from eslint

Reply:
ESLint throws a warning on this because ESLint is confusing this class with React component:
```
class ManifestTooLarge {
  static displayName = 'ManifestTooLarge'
}
```

Reply:
ok, noted, but this is used in our base error class, so changing it throughout the app is quite involved. btw i didnt see that warning, maybe because warnings dont abort commit, and i only run eslint as a commit hook)